opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -418,11 +418,14 @@ export const roleEditContext = async (context, user, roleId, input) => {
   });
 };
 
+const isAdministratedOrga = (user, organizationId) => {
```

Comment:
`isUserAdministratingOrga` ? I think we miss the important part that we want this orga to be administrated by this current user, not a random one